# Message personnalisÃ© pour la position des robots
string name
float32 x
float32 y
float32 theta
string status
